### DISCUSSION TITLE
Unnecessary Error Message Prefix in API Responses

### DISCUSSION THREAD

**Discussion by takhyun12** [bentoml/BentoML#4909] [Q&A]
Hello,

I am using `bentoml.exceptions` such as `UnprocessableEntity` in the following way when an error occurs in the `service.api`:

```python
@service.api(
    input=Multipart(data=File()),
    output=Image(),
    route="/inpaint/v1/wrinkles",
)
async def api(data: BytesIO):
    (...)
    if not face_masks:
        raise UnprocessableEntity(message="No face detected")
```

However, I noticed that the error message appears in the following format:

```
"BentoService error handling API request: No face detected"
```

I believe that exposing the serving framework in the error message is unnecessary, and I would like to remove the prefix such as `BentoService error handling API request`. I only want the JSON-formatted message to be returned to the client.

I have tried various approaches mentioned in the [BentoML exceptions documentation](https://docs.bentoml.org/en/latest/reference/exceptions.html) but have not found a way to achieve this. It seems like it might be due to the way it's implemented in the following [code](https://github.com/bentoml/BentoML/blob/66adb62ea0faa876ddd8ee3c050d777f3f909b4b/bentoml/server/bento_api_server.py#L331).

Could you please advise if there is a way to resolve this issue?

Thank you in advance for your assistance!

**Comment by aarnphm** [comment]
This is mainly the server log. If you want to return a JSON response to the client they it is probably better to return a JSONResponse, or a pydantic model for error
